_config.yml

```diff
@@ -53,15 +53,10 @@ gems:
 markdown: kramdown
 kramdown:
   input: GFM
-  syntax_highlighter: rouge
+  syntax_highlighter: none
```

Comment:
I'd strongly recomment keeping `rouge` enabled. This is highlighting the code on the site as part of the build, rather than client-side. This means that JS is not required at all. We only really need the JS version for the package details page, everything else can do build-time highlighting.

Reply:
Sure!